troposphere/static/js/components/images/detail/tags/Tags.react.js

```diff
@@ -4,9 +4,14 @@ import Tag from "components/common/tags/Tag.react";
 
 export default React.createClass({
     displayName: "Tags",
-
+    // We are calling first() on some of our collections,
```

Comment:
Please track this down. The proptypes help us detect when we are being inconsistent, so that we can fix it, not make the proptypes more relaxed. I saw you fixed it in one place in the `slice` below.